db-testsuite/src/index.ts

```diff
@@ -1,23 +1,17 @@
 import anyTest, { TestInterface } from 'ava';
-import { mkdtemp } from 'fs';
-import rmrf from 'rmfr';
-import { tmpdir } from 'os';
-import { promisify } from 'util';
-import * as path from 'path';
 import { AddressInfo } from 'net';
 import Koa from 'koa';
 import KoaRouter from 'koa-router';
 import { createServer, Server } from 'http';
-import { Handler } from '@binaris/shift-leveldb-server';
-import { DBRouter } from '@binaris/shift-interfaces-koa-server';
-import { DB } from '../../db';
+import { DBRouter, DBHandler } from '@binaris/shift-interfaces-koa-server';
+import { DB } from '@binaris/shift-db/dist/db';
 
 interface Context {
-  dbDir: string;
   client: DB;
+  injectedContext: any;
 }
 
-const test = anyTest as TestInterface<Context>;
+export const test = anyTest as TestInterface<Context>;
```

Comment:
Extra `export`?

Reply:
Nope, you can add `test.before` and other hooks this way.

Reply:
Used by thoth.